gprofiler/profilers/node.py

```diff
@@ -205,13 +206,15 @@ def _copy_module_into_process_ns(process: psutil.Process, musl: bool, version: s
 def _generate_perf_map(module_path: str, nspid: int, ns_link_name: str) -> None:
     sock = create_debugger_socket(nspid, ns_link_name)
     _change_dso_state(sock, module_path, "start")
+    logger.info(f"Changed DSO state to start for pid: {nspid} (nodejs)")
```

Comment:
I think that logging the resolved `module_path` can also help (i.e pass it as an extra to the `debug(..)` call)